macos/FluentUI/Button/Button.swift

```diff
@@ -597,6 +598,16 @@ private struct ButtonSizeParameters {
 		titleToImageSpacing: 10,
 		titleToImageVerticalSpacingAdjustment: 7
 	)
+	
+	static let medium = ButtonSizeParameters(
+		fontSize: 13,  // line height: 17
+		cornerRadius: 6,
+		verticalPadding: 2.0,  // overall height: 24
```

Comment:
Somethings not right. These measured perfectly on my local build in the OfficeSpaceTestApp, but in the FluentUITest app the size is actually 20pts high (same as the small size). Need to investigate why that might be the case.

Reply:
Looks like it's the images being used that are pushing the size from 20 to 24pt.

Reply:
Looks like we just need to adjust the "verticalPadding" from 2.0 to 4.0.

Reply:
Hmm, if you make the above change, then the buttons in the OfficeSpaceTestApp are 28pts tall.

Reply:
Button height grows to fit the images but also decreases when smaller images are used, so we may want to add a minimum height constraint.